components/Home/Bonding/requests.jsx

```diff
@@ -48,6 +48,15 @@ const LP_PAIRS = {
     poolId:
       '0x62309056c759c36879cde93693e7903bf415e4bc000200000000000000000d5f',
   },
+  // arbitrum
+  '0xf9825A563222f9eFC81e369311DAdb13D68e60a4': {
```

Comment:
```suggestion
  '0x36B203Cb3086269f005a4b987772452243c0767f': {
```

Reply:
please also replace '0x06512E620A8317da51a73690A596Aca97287b31D'  in line 43 with '0xf9825A563222f9eFC81e369311DAdb13D68e60a4'